TYPESCRIPT.md

```diff
@@ -3,51 +3,64 @@
+   points if you write a PR yourself to the EUI repo to add the types, but having them available back in Kibana will
+   take some time, as a new EUI release will need to be generated, then that new release pointed to in Kibana. Best, to
+   make forward progress, to do a temporary workaround.
 
 ```ts
 // typings/@elastic/eui/index.d.ts
 
 declare module '@elastic/eui' {
   // Add your types here
   export const EuiPopoverTitle: React.FC<EuiPopoverTitleProps>;
-  ...
+...
 }
 ```
 
 ```ts
 // you can now import it in <your-plugin-file.ts>
 
-import { EuiPopoverTitle } from '@elastic/eui';
+import {EuiPopoverTitle} from '@elastic/eui';
 ```
 
 Some background on the differences between module declaration and augmentation:
 
-In TypeScript module declarations can not be merged, which means each module can only be declared once. But it is possible to augment previously declared modules. The documentation about the distinction between module declaration and augmentation is sparse. The observed rules for `declare module '...' {}` in a `.d.ts` file seem to be:
+In TypeScript module declarations can not be merged, which means each module can only be declared once. But it is
+possible to augment previously declared modules. The documentation about the distinction between module declaration and
+augmentation is sparse. The observed rules for `declare module '...' {}` in a `.d.ts` file seem to be:
 
 * it is treated as a module declaration when the file itself is not a module
 * it is treated as a module augmentation when the file itself is module
 
-A `.d.ts` file is treated as a module if it contains any top-level `import` or `export` statements. That means that in order to write a module declaration the `import`s must be contained within the `declare` block and none must be located on the topmost level. Conversely, to write a module augmentation there must be at least one top-level `import` or `export` and the `declare` block must not contain any `import` statements.
+A `.d.ts` file is treated as a module if it contains any top-level `import` or `export` statements. That means that in
+order to write a module declaration the `import`s must be contained within the `declare` block and none must be located
+on the topmost level. Conversely, to write a module augmentation there must be at least one top-level `import`
+or `export` and the `declare` block must not contain any `import` statements.
 
-Since `@elastic/eui` already ships with a module declaration, any local additions must be performed using module augmentation, e.g.
+Since `@elastic/eui` already ships with a module declaration, any local additions must be performed using module
+augmentation, e.g.
 
 ```typescript
 // file `typings/@elastic/eui/index.d.ts`
 
-import { CommonProps } from '@elastic/eui';
-import { FC } from 'react';
+import {CommonProps} from '@elastic/eui';
+import {FC} from 'react';
```

Comment:
These spaces should stay.

eslint won't allow destructured objects without spaces after the curly braces, so better to keep the example as-is, since this is the way devs will be required to write it in Kibana.

```suggestion
import { CommonProps } from '@elastic/eui';
import { FC } from 'react';
```

Reply:
fixed